styles.css

```diff
@@ -736,3 +742,257 @@ ul {
+  flex-direction: column;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  justify-content: center;
+  font-size: 30px;
+  align-items: center;
+  align-content: center;
+  width: 327px;
+  max-height: 600px;
+  margin: 20px;
+}
+
+.mobile-button-pop-up {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-content: space-between;
+  width: 100%;
+}
+
+.mobile-pop-up-item-image {
+  width: 100%;
+  height: 100%;
+}
+
+.mobile-pop-up-container-image {
+  width: 90%;
+  height: 252px;
+  background: #ebf0ee;
+  margin: 20px;
+}
+
+.mobile-pop-up-container-title h2 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  text-align: left;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  width: 70%;
+  margin: 0 10px 30px 20px;
+}
+
+.mobile-pop-up-container-projects {
+  display: flex;
+  min-height: 40px;
+  flex-direction: row;
+  align-items: flex-start;
+  margin-left: 10px;
+}
+
+.mobile-pop-up-container-projects-element {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  min-width: 45px;
+}
+
+.mobile-pop-up-text {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  text-align: left;
+  font-weight: 600;
+  margin: 0 10px 30px 20px;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.mobile-pop-up-container-btn {
+  display: flex;
+  flex-flow: row;
+  min-height: 40px;
+  width: 150px;
+  border-radius: 4px;
+  background: #36b37f;
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  align-items: center;
+  font-weight: 500;
+  font-size: 15px;
+  box-shadow: none;
+  transition: box-shadow 2s;
+}
+
+.mobile-pop-up-container-btn:first-child {
+  margin: 15px 2px 20px 16px;
+}
+
+.mobile-pop-up-container-btn:last-child {
+  margin: 15px 16px 20px 2px;
+}
+
+#modal-hiden-image {
+  display: flex;
+  align-content: flex-end;
+  margin-right: 0;
+  margin-top: 10px;
+  margin-left: 85%;
+  background: none;
+  color: #fff;
+  text-decoration: none;
+}
+
+.mobile-pop-up-container-projects-element li {
+  display: flex;
+  justify-content: flex-start;
+  align-items: flex-start;
+  font-family: 'Inter', sans-serif;
+  font-size: 12px;
+  font-weight: 600;
+  color: #3a4a42;
+  background-color: #ebf0ee;
+  min-height: 20px;
+  min-width: 10px;
+  border-radius: 4px;
+  margin: 0 5px 0 5px;
+  padding: 6px 10px 1px 10px;
+}
+
+.hidden-mobile-button-pop-up {
+  display: none;
+}
+
+.modal-social {
+  margin: 0 10px 0 10px;
+}
+
+@media screen and (min-width: 991px) {
+  #mobile-pop-up {
+    position: fixed;
+    z-index: 1;
+    top: 0;
+    left: 0;
+    background-color: #fff;
+    display: none;
+    flex-direction: column;
+    border: 1px solid #d0d9d4;
+    border-radius: 8px;
+    justify-content: center;
+    font-size: 30px;
+    align-items: center;
+    align-content: center;
+    width: 60%;
+    margin-left: 250px;
+  }
+
+  .hidden-mobile-button-pop-up {
+    display: flex !important;
```

Comment:
- [ ] **[OPTIONAL]** Having the `!important;` tag here is not necessary as it is a bad practice to use the tag. Let's try to remove it from our codebase and see that all things still work without it. I suggest you remove it in order to achieve good code practice.